engine/src/main/java/org/terasology/engine/persistence/typeHandling/RegisterTypeHandler.java

```diff
@@ -11,8 +11,8 @@
 import java.lang.annotation.Target;
 
 /**
- * Marks a {@link TypeHandler} to be automatically registered at a
- * {@link TypeHandlerLibrary TypeHandlerLibrary} on environment change.
+ * Marks a {@link org.terasology.persistence.typeHandling.TypeHandler} to be automatically registered at a
```

Comment:
Is this really necessary? I thought that JavaDoc would be smart enough to reuse imports and allow for shorthand links within the same package 🤔 How does this render?

Reply:
javadoc complained about an unknown reference and auto-generated the long version when I re-added it :woman_shrugging:

Reply:
how do you "auto-generate" something with javadoc?

Reply:
Generally in an IDE like IntelliJ if you start by entering `/**` atop a method then IntelliJ will generate the whole Javadoc block (if you shift-enter or something?) including defined parameters etc. There are probably similar tricks when you have other interactions with Javadoc. 

Plus the whole copilot madness which can just be eerily close to generating a pile of stuff correctly or oh-so-far from it at times :-)